### DISCUSSION TITLE
Adding an unbranded universal remote control or adding a remote control for PHILIPS TVs

### DISCUSSION THREAD

**Discussion by lukazareba** [PRProd/HA-Firemote#316] [New Device Family Support Request]
### Is your feature request related to a problem?

Mam telewizor PHILIPS z android tv, w bazie pilotów działają wszystkie ale brakuje mi pilota dedykowanego do PHILIPS, czy można prosić o jego dodanie?

### Feature Request

I have a PHILIPS TV with android tv, in the remote control database they all work but I am missing a remote dedicated to PHILIPS, can you please add it?
![image](https://github.com/PRProd/HA-Firemote/assets/116034829/540d5adc-45d2-4de4-b288-12ec672d110f)
![image](https://github.com/PRProd/HA-Firemote/assets/116034829/ceb147cb-1b8a-452c-9db0-e2d6f4995c80)

**Comment by PRProd** [comment]
Greetings @lukazareba!

Your request falls under the category of **New Device Support**.  The way these requests are handled is through a [poll in Discussion #41](https://github.com/PRProd/HA-Firemote/discussions/41) where the community gets to help decide what gets added to Firemote next.  I recommend going over there instead.

Quick question: Do you speak a language other than English?  I think I saw Polish in this request.  If so, would you be willing to help out the project by opening a New Language Support request? [Click here](https://github.com/PRProd/HA-Firemote/issues/new?assignees=PRProd&labels=Region+Issue%2Cenhancement&projects=&template=New_Language_Support.yaml&title=%5BNew+Language+Support%5D%3A+Polish&langcode=pl) if you are able to help out.

**Comment by PRProd** [comment]
Thank you for your willingness to help with your [Polish language support request](https://github.com/PRProd/HA-Firemote/issues/315)!

Would you be able to provide some additional information about this device?  Product links, model numbers, etc. would all be very helpful.  Thanks.
